model/ftn/w3agcmmd.ftn

```diff
@@ -120,7 +120,7 @@
          ! Ocean sea surface current (m.s-1) (u-component)
          ! ---------------------------------------------------------------------
          IF (SND_FLD(IB_DO)%CL_FIELD_NAME == 'WW3_WSSU') THEN
-            TMP(1:NSEALM) = 0.0 
+            TMP(1:NSEAL) = 0.0 
             DO JSEA=1, NSEAL
```

Comment:
Maybe you should also rewrite it as :
WHERE(CX(1:NSEAL) /= UNDEF) TMP(1:NSEAL)=CX(1:NSEAL)

Reply:
> Maybe you should also rewrite it as :
> WHERE(CX(1:NSEAL) /= UNDEF) TMP(1:NSEAL)=CX(1:NSEAL)

I do not think a WHERE sentence will work in this case, as CX and CY are vectors of dimension 1:NSEA. The reason why the loop is done explicitly is to find the right index for this vector for a given processor:

`ISEA=IAPROC+(JSEA-1)*NAPROC`

Notice that the same approach is followed for the variable ICEF in w3igcmmd.f90, which also has dimensions 1:NSEA.

Reply:
right ! BTY the line 139 in w3agcmd.ftn since it force TMP to be filled by CY after the test on UNDEF

Reply:
> right ! BTY the line 139 in w3agcmd.ftn since it force TMP to be filled by CY after the test on UNDEF

Agree, that line should be deleted.